src/css/rooms.scss

```diff
@@ -0,0 +1,93 @@
+@import 'variables';
+
+.my-rooms {
+  font-size: 1.5rem;
+}
+
+// ***** filter rooms *****
+.checkbox-container {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.fa-filter {
+  padding-right: 15px;
+  font-size: 1.5rem;
+}
+
+.filter-type-container {
+  white-space: nowrap;
+}
+
+.room-type-label {
+  padding-left: 4px;
+  padding-right: 15px;
+  font-size: 1.15rem;
+}
+
+.apology-message {
+  color: $accent-dark;
+  font-size: 1rem;
+}
+
+
+// ***** display available rooms *****
+.available-rooms {
+  margin-top: 40px;
+  margin-bottom: 20px;
+  font-size: 1.5rem;
+}
+
+.rooms-container {
+  margin-top: 40px;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  // grid-template-rows: repeat(2, 1fr);
+  grid-gap: 15px;
+}
+
+.room {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  padding: 5px;
+  min-height: 150px;
+  min-width: 300px;
+  background-color: white;
+  border-radius: 5px;
+  box-shadow: 0px 0px 7px $main-medium;
+
```

Comment:
if you need more variables to check off on the specs then you might be able to make your box-shadow a var or even a mixin. just make the parameter a different color if you want the shadow to be dynamic. (i completely forgot about making mixins dynamic and parameters until trravis reminded me about them in my eval)

Reply:
Great idea, thanks!!